docs/Using_the_EasyBuild_command_line.rst

```diff
@@ -532,6 +542,7 @@ Note how the different status symbols denote distinct handling states by EasyBui
 * ``[ ]`` The build is not available, EasyBuild will deliver it
 * ``[x]`` The build is available, EasyBuild will skip building this module
 * ``[F]`` The build is available, however EasyBuild has been asked to force a rebuild and will do so
+* ``[R]`` The build is available, and the application will be rebuilt as request by rebuild flag
```

Comment:
under the condition that https://github.com/hpcugent/easybuild-framework/pull/1435 is updated :)